app/components/participation-list.js

```diff
@@ -37,13 +51,14 @@ export default class ParticipationListComponent extends Component {
     return sorted;
   }
   get mandateesNotPresent() {
-    if(this.aanwezigenBijStart.length > 0 && this.possibleParticipants) {
-      const aanwezigenUris = this.aanwezigenBijStart.map((mandataris) => mandataris.uri);
-      const notPresent = this.possibleParticipants.filter((mandataris) => !aanwezigenUris.includes(mandataris.uri));
-      const sorted=notPresent.sortBy('isBestuurlijkeAliasVan.achternaam');
-      return sorted;
-    }
-    return [];
+    // if(this.aanwezigenBijStart.length > 0 && this.possibleParticipants) {
+    //   const aanwezigenUris = this.aanwezigenBijStart.map((mandataris) => mandataris.uri);
+    //   const notPresent = this.possibleParticipants.filter((mandataris) => !aanwezigenUris.includes(mandataris.uri));
+    //   const sorted=notPresent.sortBy('isBestuurlijkeAliasVan.achternaam');
+    //   return sorted;
+    // }
```

Comment:
Why we comment out this part of the code? If it's not needed I would remove it

Reply:
Because I forgot :stuck_out_tongue: